### ISSUE TITLE
Bilingual pages site-related pages (Privacy, About, Terms) have english titles / url

### ISSUE THREAD

**Issue by Phanoix** [gctools-outilsgc/gcconnex#18]
Bilingual pages site-related pages (Privacy, About, Terms) should have bilingual titles.

Relevant files I've dug up:
/mod\externalpages\languages\  ,
 /mod\externalpages\start.php, externalpages\views\default\expages\menu.php
